src/main/java/cz/muni/pa165/teamwhite/formula1/DriverDao.java

```diff
@@ -0,0 +1,23 @@
+package cz.muni.pa165.teamwhite.formula1;
+
+import cz.muni.pa165.teamwhite.formula1.entity.Driver;
+
+import javax.persistence.Entity;
+import java.util.List;
+
+
+/**
+ * @author Jakub Fajkus
+ */
+@Entity
+public interface DriverDao {
+    void create(Driver d);
+
+    Driver update(Driver d);
+
+    List<Driver> findAll();
+
+    Driver findById(Long id);
+
```

Comment:
možná bych přidala findByName / findBySurname ... ?

Reply:
a nechceme hledat jezdce i podle nejakych vlastnosti? jen dotaz :smiley:

Reply:
na to bych kaslal.. nemas jistotu, ze tam takovy jezdec je... z implementacniho hlediska bych to pozdeji videl tak, ze se Managerovi ukaze seznam jezdcu s checkboxy :-)

Reply:
Az budu potrebovat hledat jezdce podle jmena nebo vlastnosti, tak si to naprogramuju... :smile: